.gitignore

```diff
@@ -22,3 +22,11 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+/test-results/
+/playwright-report/
+/blob-report/
+/playwright/.cache/
+/test-results/
+/playwright-report/
+/blob-report/
+/playwright/.cache/
```

Comment:
these are duplicated

Reply:
Yeah, I've run two inits because of trying to make it work on my windows' wsl and playwright duplicated them

Reply:
done